docs/core/preview3/tools/dotnet-new.md

```diff
@@ -4,56 +4,103 @@ description: The dotnet-new command creates new .NET Core projects in the curren
 
 ## Synopsis
-`dotnet new [--help] [--type] [--lang]`
+```
+dotnet new [template] [-lang|--language] [-n|--name] [-o|--output]
+dotnet new [template] [-l|--list]
+dotnet new [-all|--show-all]
+dotnet new [-h|--help]
+```
 
 ## Description
-The `dotnet new` command provides a convenient way to initialize a valid .NET Core project and sample source code to try out the Command Line Interface (CLI) toolset. 
-
-This command is invoked in the context of a directory. When invoked, the command will result in two main artifacts being dropped to the current directory: 
+The `dotnet new` command provides a convenient way to initialize a valid .NET Core project and sample source code to try out the Command-line interface (CLI) toolset. 
 
-1. A `Program.cs` (or `Program.fs`) file that contains a sample "Hello World" program.
-2. A valid csproj project file.
+This command is invoked in the context of a directory. When invoked, the command will scaffold out the resources and files according to the template and options passed into the command. 
 
 After this, the project is ready to be compiled and/or edited further. 
 
+## Arguments
+template - The template to instansiate when the command is invoked.
+
+The command contains a default list of templates; use `dotnet new --help`. 
+
 ## Options
 
+`-l|--list`         
+
+List templates containing the specified name.
+
+`-lang|--language`  
+
+Specifies the language of the template to create
+
+`-n|--name`         
+
+The name for the output being created. If no name is specified, the name of the current directory is used.
+
+`-o|--output`       
+
+Location to place the generated output.
+
+`-all|--show-all`   
+
+Shows all templates for a specific type of project.
+
 `-h|--help`
 
-Prints out a short help for the command.  
+Prints out help for the command.
+
+## Template options
+Each project template may have additional options available. The core templates, for example, have the following.
+
+**console, xunit, mstest**
+
+`-f|--framework` - Specifies which framework to target. Values: netcoreapp1.0 or netcoreapp1.1 (Default: netcoreapp1.0)
+
+**web, webapi**
 
-`-l|--lang C#`
+`-f|--framework` - Specifies which framework to target. Values: netcoreapp1.0 or netcoreapp1.1 (Default: netcoreapp1.0)
+ 
+**mvc**
 
-Language of the project. Defaults to `C#`. Other valid values are `csharp` and `cs`.
+`-f|--framework` - Specifies which framework to target. Values: netcoreapp1.0 or netcoreapp1.1 (Default: netcoreapp1.0)
 
-`-t|--type`
+`-au|--authentication` -  The type of authentication to use. Values: None or Individual (Default: None)
 
-Type of the project. Valid values for C# are `console`, `web`, `lib` and `xunittest` and for F# only `console` is valid. 
+`-uld|--use-local-db` - Whether or not to use LocalDB instead of SQLite. Values: true or false (Default: false)
+
+**classlib**
+
+`-f|--framework` - Specifies which framework to target. Values: netcoreapp1.0 - 1.6 (Default: netcoreapp1.4).
```

Comment:
it accepts netcoreapp1.0 or netcoreapp1.1 and netstandard1.0 - 1.6 and default is netstandard1.4